app/importers/merchants_importer.rb

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+
+class MerchantsImporter
+  attr_reader :filename,
+              :number_of_lines
+
+  def initialize(filename)
+    @filename = filename
+    @number_of_lines = `wc -l #{filename}`.to_i
+  end
+
+  def import
+    bar = ProgressBar.create(title: "Merchants", total: number_of_lines)
+
+    CSV.foreach(filename, headers: true) do |row|
+
+      merchant_attributes = {
+        id:         row['id'].to_i,
+        name:       row['name'],
+        updated_at: to_date(row['updated_at']),
+        created_at: to_date(row['created_at'])
+      }
+            
+      Merchant.create!(merchant_attributes)
```

Comment:
I dig it.  The data we're importing is fairly simple - I like your decision to use a hash for the attributes when creating a new Merchant.  

I see what you're saying about `create` vs `create!`.  This feels like a more prudent approach, which I like.  Perhaps once we have the individual importers written, we also create a `rake import:all` task which starts by cleaning the db.  Thoughts?

Also, since we've nicely extracted the importer to a class, it would be great to write a spec for it.  I don't think the spec has to be anything fancy, just present it with a few rows from the merchants.csv and make sure that the correct number of Merchants get created in the database.  How does that sound to you?